cg/meta/backup/backup.py

```diff
@@ -405,14 +405,15 @@ def mark_file_as_archived(self, spring_file_path: Path) -> None:
             return
         hk_spring_file: File = self.hk_api.files(path=str(spring_file_path)).first()
         LOG.info(f"Setting {spring_file_path} to archived in Housekeeper")
-        self.hk_api.set_to_archive(file=hk_spring_file, value=True)
+        if hk_spring_file:
+            self.hk_api.set_to_archive(file=hk_spring_file, value=True)
 
     def is_to_be_retrieved_and_decrypted(self, spring_file_path: Path) -> bool:
```

Comment:
The name of this function is confusing since we don't actually check whether the file needs to be decrypted.